releasenotes/notes/fix-kafka-consume-dispatch-48cd5912b1282ab9.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    kafka: This fix resolves an issue where only the first message in a batch was dispatched to Data Streams Monitoring (DSM) when consuming multiple Kafka messages
```

Comment:
not sure I understand this fix, we are fixing an issue with the `kafka` integration while introducing a new integration?